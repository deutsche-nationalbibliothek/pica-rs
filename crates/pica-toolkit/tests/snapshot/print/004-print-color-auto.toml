bin.name = "pica"
args = "print --color=auto"
status = "success"
stdin = "003@ \u001f0123456789X\u001e\n"
stderr = ""
