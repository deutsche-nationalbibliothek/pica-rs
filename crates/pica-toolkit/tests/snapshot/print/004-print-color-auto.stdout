003@ $0 123456789X

